src/utils/changelog.ts

```diff
@@ -288,31 +469,61 @@ export async function generateChangesetFromGit(
       continue;
     }
 
-    const commit = {
+    const labels = githubCommit?.labels ?? [];
+    const author = githubCommit?.author;
+
+    // Apply global exclusions
+    if (shouldExcludePR(labels, author, releaseConfig)) {
+      continue;
+    }
+
+    // Apply category-level exclusions (these completely hide the PR)
+    if (isCategoryLevelExcluded(labels, author, releaseConfig)) {
+      continue;
+    }
```

Comment:
As mentioned earlier, this logic is incorrect. This just hides the PR from a specific category, not necessarily from everything else.